components/nextdoor/actions/create-ad-group/create-ad-group.mjs

```diff
@@ -0,0 +1,202 @@
+import app from "../../nextdoor.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "nextdoor-create-ad-group",
+  name: "Create Ad Group",
+  description: "Creates an ad group based on the input payload for an existing campaign. [See the documentation](https://developer.nextdoor.com/reference/adgroup-create).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    advertiserId: {
+      propDefinition: [
+        app,
+        "advertiserId",
+      ],
+    },
+    campaignId: {
+      propDefinition: [
+        app,
+        "campaignId",
+        ({ advertiserId }) => ({
+          advertiserId,
+        }),
+      ],
+    },
+    name: {
+      description: "The name of the ad group.",
+      propDefinition: [
+        app,
+        "name",
+      ],
+    },
+    placements: {
+      type: "string[]",
+      label: "Placements",
+      description: "The placements for the ad group.",
+      options: [
+        "RHR",
+        "FEED",
+        "FSF",
+      ],
+    },
+    bidAmount: {
+      type: "string",
+      label: "Bid Amount",
+      description: "The bid amount for the ad group. The value must be a string in the format `USD 10` as an example.",
+    },
+    bidPricingType: {
+      type: "string",
+      label: "Bid Pricing Type",
+      description: "The bid pricing type for the ad group. The value must be one of `CPM`.",
+      options: [
+        "CPM",
+      ],
+    },
+    budgetAmount: {
+      type: "string",
+      label: "Budget Amount",
+      description: "The budget amount for the ad group. The value must be a string in the format `USD 10` as an example.",
+    },
+    budgetType: {
+      type: "string",
+      label: "Budget Type",
+      description: "The budget type for the ad group.",
+      options: [
+        "DAILY_CAP_MONEY",
+      ],
```

Comment:
Is there really only one option? If so, it should probably be set in the code directly instead of being a prop